cmd/ketch/output/output.go

```diff
@@ -0,0 +1,37 @@
+package output
+
+import (
+	"io"
+
+	"github.com/spf13/pflag"
+)
+
+type Writer interface {
+	Write() error
+}
+
+func Write(data interface{}, out io.Writer, flags *pflag.FlagSet) error {
```

Comment:
can this function take in `outputFlag string`? 
It's a utility function and there is no need to share information about the flagset.

Reply:
Good call. Change made.